pegasus/sites.v3/code.org/public/pd-workshop-survey/splat.haml

```diff
@@ -0,0 +1,374 @@
+---
+title: "Professional Development Workshop Survey"
+---
+:ruby
+  dont_cache
+  enrollment_code = request.splat_path_info[1..-1]
+  enrollment = DASHBOARD_DB[:pd_enrollments].where(code: enrollment_code).first
+  pass unless enrollment
+
+  workshop = DASHBOARD_DB[:pd_workshops].where(id: enrollment[:pd_workshop_id]).first
+  pass unless workshop
+
+  plp = DASHBOARD_DB[:professional_learning_partners].where(contact_id: workshop[:organizer_id]).first
+  # Have they ever taken any pd-workshop-survey?
+  is_first_survey = DB[:forms].where(kind: 'PdWorkshopSurvey').count == 0
+
+  # Have they take this particular survey?
+  previous_response = DB[:forms].where(kind: 'PdWorkshopSurvey', source_id: enrollment[:id]).first
+
+  course = workshop[:course]
+  subject = workshop[:subject]
+
+  def local_view(name, *params)
+    view "pd_workshop_survey/#{name.to_s}", *params
+  end
+
+%script{type: "text/javascript", src: "/js/sifter.min.js"}
+%script{type: "text/javascript", src: "/js/microplugin.min.js"}
+%script{type: "text/javascript", src: "/js/selectize.min.js"}
+%script{type: "text/javascript", src: "/js/selectize-fast-click.js"}
+%link{rel: "stylesheet", type: "text/css", href: "/css/selectize.bootstrap3.css"}/
+
+- if previous_response
+  %h1 Thank you.
+  %p You have already completed this survey.
+
+- else
+
+  %form#pd-workshop-survey-form{role: 'form'}
+    %input{name: 'enrollment_id_i', type:'hidden', value: enrollment[:id]}
+    %input{name: 'workshop_id_i', type: 'hidden', value: workshop[:id]}
+    %input{name: 'plp_b', type: 'hidden', value: plp.nil? ? '0' : '1'}
+    %input{name: 'include_demographics_b', type: 'hidden', value: is_first_survey ? '1' : '0'}
+
+    %h1
+      Satisfaction Survey for Code.org's
+      = course
+      = subject
+      Professional Development Workshop
+
+    #error-message{style: 'display: none'}
+
+    .main-section
+      .form-group
+        = local_view :yes_no_select,
+          name: 'consent_b',
+          label: 'Your participation is voluntary but strongly encouraged. Do you consent to completing this survey?'
+
+    .consent-only
+      .form-group
+        = local_view :yes_no_select,
+          name: 'will_teach_b',
+          label: "Will you be teaching/are you teaching #{course} #{subject} in the 2016-17 school year?"
+        %div{id: 'will-teach-no-explain-wrapper', style: 'display: none;'}
+          %label.control-label{for: 'will_not_teach_explanation_s'}<
+            %em{style: 'font-weight: normal;'} Please explain why not.
+          %div
+            %input.form-control{name: 'will_not_teach_explanation_s', placeholder: 'Why not?', type: 'text'}
+
+      .form-group
+        = local_view :multi_select,
+          name: 'reason_for_attending_ss[]',
+          label: 'Reason for attending? (select all that apply)'
```

Comment:
Even though we've used this kind of multi-select in the past, we've seen a lot of users struggle using this. It's so subtle a difference that it's multi-select compared to the single select. Only visual difference is the little black triangle and the gray box around the selected items. 

I think using selectize to type and search your selections is most valuable when the list is very long. But since all the multi-select question in this survey are short, can we use checkboxes instead? I think this is more recognizable and usable.